src/kiota/Program.cs

```diff
@@ -14,7 +15,19 @@ static async Task<int> Main(string[] args)
         var rootCommand = KiotaHost.GetRootCommand();
         var parser = new CommandLineBuilder(rootCommand)
             .UseDefaults()
-            .UseHost(static args => Host.CreateDefaultBuilder(args).ConfigureKiotaTelemetryServices())
+            .UseHost(static args =>
+            {
+                return Host.CreateDefaultBuilder(args)
+                    .ConfigureKiotaTelemetryServices()
+                    .ConfigureLogging(static logging =>
+                    {
+                        logging.ClearProviders();
```

Comment:
Doesn't that mean that CLI users won't even get error messages? Or the requested log level?

Reply:
This logger isn't what kiota actually uses. Each command [sets up its own logger](https://github.com/microsoft/kiota/blob/7976e7ef351863529161c64ff9746a1e6db642d8/src/kiota/Handlers/BaseKiotaCommandHandler.cs#L136). We still haven't done the work of centralizing the logging infrastructure. I could also clear providers and the CLI would still work.

Reply:
thanks for the context here. Let's leave that cleanup to another day so we can unblock the release and our partners